dfttopif/drivers.py

```diff
@@ -195,12 +191,28 @@ def directory_to_pif(directory, verbose=0, quality_report=True, inline=True):
         chem.properties.append(prop)
 
     # Check to see if we should add the quality report
-    if quality_report and isinstance(parser, VaspParser) :
-        _add_quality_report(directory, chem)
+    if quality_report and isinstance(parser, VaspParser):
+        _add_quality_report(parser, chem)
 
     return chem
 
-def convert(files=[], **kwargs):
+
+def directory_to_pif(directory, **kwargs):
+    """
+    Convert a directory to a pif
+    :param directory: Directory to convert to a pif
+    :param kwargs: any additional keyword arguments. (See `files_to_pif`)
+    :return: the created pif
+    """
+
+    # Get the files
+    files = [os.path.join(directory, f) for f in os.listdir(directory)]
```

Comment:
Why not add the `if os.path.isfile(f)` filter here?

Reply:
Any objections to just adding it to the initializer for DFTParser?

Reply:
Oops, I misunderstood your comment. I put the filter in a different loop because I didn't want to call `os.path.join` twice in the same loop. No other considerations behind it. I'll change it